internal/database/mysql_test.go

```diff
@@ -195,29 +206,31 @@ func TestAuthenticateLogin(t *testing.T) {
 	subject, probe := setup(t)
 	defer teardown(t, subject, probe)
 
-	happyUser := "testUser"
+	happyEmail := "test_user@example.com"
 	happyPass := "testPass"
-	if err := subject.AddUser(happyUser, "test_user@example.com", happyPass); err != nil {
-		t.Fatal("Failed to add", happyUser, "-", err)
+	_, err := subject.AddUser("test", "", "user", "test_user@example.com",
+		happyPass, "2000-02-13")
+	if err != nil {
```

Comment:
We should add some test cases for users with weir names (spaces and stuff). We really also *need* to create these:
- Fail to add user with empty first name
- Fail to add user with empty last name
- User with empty middle name is actually stored as "nil" (use sql.NullString)
- Users with same names, all works good.

Reply:
I didn't use " sql.NullString", but I added testing case for middle name.